RedHatInsights/insights-inventory-frontend
# chore(ESSNTL-4219): Run with mocked API server

Implements https://issues.redhat.com/browse/ESSNTL-4219.

This provides another local deployment option where webpack server starts together with the mocked API server. The API server is built on top of OpenAPI schema.

prism CLI docs: https://docs.stoplight.io/docs/prism/674b27b261c3c-overview

## How to test:

1. Check updated README.md and follow the steps.
2. While deployed locally, verify that HBI API endpoints are redirected to local prism server.
